configs/config.json.cluster.example

```diff
@@ -22,23 +22,6 @@
         "type": "HandleSoftBan"
       },
       {
-        "type": "CompleteTutorial",
-        "config": {
-          "enabled": false,
-          "// set a name": "",
-          "nickname": "",
-          "// 0 = No Team, 1 = Blue, 2 = Red, 3 = Yellow": "",
-          "team": 0
-        }
-      },
-      {
-        "type": "CollectLevelUpReward",
-        "config": {
-          "collect_reward": true,
-          "level_limit": -1
```

Comment:
This can help a bot stop at a certain level. Would need to be readded somewhere.